pkg/kubectl/cmd/config/view.go

```diff
@@ -99,7 +99,23 @@ func NewCmdConfigView(f cmdutil.Factory, out, errOut io.Writer, ConfigAccess cli
 	return cmd
 }
 
-func (o ViewOptions) Run(out io.Writer, printer printers.ResourcePrinter) error {
+func (o *ViewOptions) Complete(cmd *cobra.Command, out io.Writer) bool {
+	if o.ConfigAccess.IsExplicitFile() {
+		if !o.Merge.Provided() {
+			o.Merge.Set("false")
+		}
+	}
+
+	o.PrintObject = func(obj runtime.Object) error {
+		return cmdutil.PrintObject(cmd, obj, out, func(p printers.ResourcePrinter) printers.ResourcePrinter {
```

Comment:
This looks off.  The flags would be bound and this either take that flag struct as an interim step and you'd call `ToPrinter` on the flags to provide the Printer interface here, right?